Packs/SecureWorks/Integrations/TaegisXDR/TaegisXDR.py

```diff
@@ -883,6 +1030,104 @@ def fetch_users_command(client: Client, env: str, args=None):
     return results
 
 
+def isolate_asset_command(client: Client, env: str, args=None):
+    if not args.get("id"):
+        raise ValueError("Cannot isolate asset, missing id")
+    if not args.get("reason"):
+        raise ValueError("Cannot isolate asset, missing reason")
+
+    variables: Dict[str, Any] = {
+        "id": args.get("id"),
+        "reason": args.get("reason")
+    }
+
+    query = """
+    mutation isolateAsset ($id: ID!, $reason: String!) {
+      isolateAsset (id: $id, reason: $reason) {
+        id
+      }
+    }
+    """
+
+    result = client.graphql_run(query=query, variables=variables)
+
+    try:
+        isolation = result["data"]["isolateAsset"]
+    except (KeyError, TypeError):
+        raise ValueError(f"Failed to isolate asset: {result['errors'][0]['message']}")
+
+    results = CommandResults(
+        outputs_prefix="TaegisXDR.AssetIsolation",
+        outputs_key_field="id",
+        outputs=isolation,
+        readable_output=tableToMarkdown(
+            "Taegis Asset Isolation",
+            isolation,
+            removeNull=True,
+        ),
+        raw_response=result,
+    )
+
+    return results
+
+
+def update_alert_status_command(client: Client, env: str, args=None):
+    if not args.get("ids"):
+        raise ValueError("Alert IDs must be defined")
+    if not args.get("status"):
+        raise ValueError("Alert status must be defined")
+
+    if args.get("status").upper() not in ALERT_STATUSES:
+        raise ValueError((
+            f"The provided status, {args['status']}, is not valid for updating an alert. "
+            f"Supported Status Values: {ALERT_STATUSES}"))
+
+    variables = {
+        "alert_ids": args.get("ids"),
+        "reason": args.get("reason", ""),
+        "resolution_status": args.get("status"),
+    }
+
+    if type(variables["alert_ids"]) == str:
+        variables["alert_ids"] = variables["alert_ids"].split(",")  # alerts://id1,alerts://id2
+    variables["alert_ids"] = [x.strip() for x in variables["alert_ids"]]  # Ensure no whitespace
```

Comment:
is `variables["alert_ids"]` should always be a list? even if we have only one id?
If so, you can just use argToList:
`"alert_ids": argToList(args.get("ids"))`
In any case, use a variable for `args.get("ids")`, as now, you change the value of a dict in runtime, and it is not a best-practice (and can cause issues)

Reply:
I had no idea about argToList. That definitely simplifies things. Thank you for the heads up.